get_data/importData.py

```diff
@@ -0,0 +1,58 @@
+from bs4 import BeautifulSoup
+import pandas as pd
+from datetime import date
+from datetime import datetime
+import csv
+import requests
+from get_data.get_data import get_current_value
+#import os
+#THIS_FOLDER = os.path.dirname(os.path.abspath(__file__))
+#file = os.path.join(THIS_FOLDER, 'macrotrends_data.csv')
+URL = "https://markets.businessinsider.com/commodities/oil-price?type=wti"
```

Comment:
Could you remove these commented out lines please if they're not being used? Also could you rename this file to use the file naming convention please, import_data.py? Thanks :)